src/lib/index.jsx

```diff
@@ -29,65 +29,64 @@ export default function AddToCalendar(WrappedButton, WrappedDropdown) {
       className: null,
       dropdownProps: {},
       items: Object.values(SHARE_SITES),
-      linkProps: {},
+      linkProps: {}
     };
 
     state = {
-      dropdownOpen: false,
+      dropdownOpen: false
     };
 
-    handleCalendarButtonClick = e => {
+    handleCalendarButtonClick = (e) => {
```

Comment:
I think you have your editor set to autoformat code based on a different set of rules. Let's not change things like this because it means different files in this repo will be out of sync. I'll add prettier and eslint configs to make this easier in the future. But for now, these are the main formatting issues I see:

* Allow dangling commas
* Don't add parenthesis around the only arg of an arrow func